src/lsp/cobol_preproc/compdir_grammar.mly

```diff
@@ -30,10 +30,11 @@
 %token NE              "<>"         [@keyword (* symbol *) "<>"]
 
 %token CDIR_DEFINE     [@keyword ">>DEFINE", "$DEFINE"]
-%token CDIR_ELIF       [@keyword ">>ELIF", ">>ELSE-IF"]
-%token CDIR_ELSE       [@keyword ">>ELSE"]
-%token CDIR_END_IF     [@keyword ">>END-IF"]
-%token CDIR_IF         [@keyword ">>IF"]
+%token CDIR_ELIF       [@keyword ">>ELIF", "$ELIF", ">>ELSE-IF", "$ELSE-IF"]
```

Comment:
Just a note: There is no `$ELIF` or `$ELSE-IF` in MF, see [their conditional compilation docs](https://www.microfocus.com/documentation/visual-cobol/vc90/DevHub/HRLHLHCOMP0J.html).
There is no `>>ELIF` or `>>ELSE-IF` in the standard (that would be a GC extension).